model/model_training/custom_datasets/__init__.py

```diff
@@ -2,7 +2,7 @@
     High level functions for model training
 """
 from custom_datasets.oasst_dataset import load_oasst_export
-from custom_datasets.prompt_dialogue import OAPrivate, PrivateInstructionTuning
+from custom_datasets.prompt_dialogue import OAPrivate  # , PrivateInstructionTuning
```

Comment:
why don't just remove it entirely, instead of leaving a comment, since the class was removed already?